modules/cnf-about-virtual-routing-and-forwarding.adoc

```diff
@@ -0,0 +1,10 @@
+[id="cnf-about-virtual-routing-and-forwarding_{context}"]
```

Comment:
Each module must include a section of the metadata indicating what assemblies the module is included in.

// Module included in the following assemblies:
//
// <List assemblies here, each on a new line>

As seen in this example template: https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc

In this case, you would list networking/multiple_networks/about-virtual-routing-and-forwarding.adoc